### ISSUE TITLE
Implement Abstract Class Example

### ISSUE THREAD

**Issue by imomin** [alexitaylor/angular-graphql-nestjs-postgres-starter-kit#32]
**Is your feature request related to a problem? Please describe.**
No, but I think this is a good starter project and having an example of abstract class would encourage developers to follow best practice.

**Describe the solution you'd like**
Just for abstracting Id, dates and active properties,  I recommend to have "common entity" folder, there create one abstract class called "base" and other "activeBase".

```
import {CreateDateColumn, UpdateDateColumn, BaseEntity, PrimaryGeneratedColumn} from 'typeorm';

export abstract class Base extends BaseEntity {
  
  @PrimaryGeneratedColumn('uuid') public id: string;
  @CreateDateColumn() public createdAt?: Date;
  @UpdateDateColumn() public updatedAt?: Date;
}
```

and 

```
import {Column} from 'typeorm';
import {Base} from "./base.entity";

export abstract class ActiveBase extends Base {
  
    @Column({ default: true })
    isActive: boolean;
}
```

Finally, User or other entities can extend ActiveBase or Base.

**Describe alternatives you've considered**

**Additional context**

What do you think?

**Comment by alexitaylor** [comment]
@imomin 
Yes, I think this is a great idea. If you want you can assign yourself this ticket and open a PR.

**Comment by imomin** [comment]
I've made the changes but i'm unable to push to changes.

**Comment by alexitaylor** [comment]
Closed with PR #37
